pkg/kapis/devops/v1alpha2/register.go

```diff
@@ -688,6 +688,9 @@ func AddJenkinsToContainer(webservice *restful.WebService, devopsClient devops.I
 		return err
 	}
 	parse.Path = strings.Trim(parse.Path, "/")
+	// this API does not belong any kind of auth scope, it should be removed in the future version
+	// see also pkg/apiserver/request/requestinfo.go
+	// please use /devops/{devops}/jenkins/{path:*} instead
 	webservice.Route(webservice.GET("/jenkins/{path:*}").
```

Comment:
please mark it as deprecated if we are going retire it

Reply:
The deprecated flag is ready. https://github.com/kubesphere/kubesphere/pull/3190/files#diff-0a2d700995b0afc6e606eae4e730b1445bf6c7b18f170c3862595b341bfd380aR693

/cc @shaowenchen 
/cc @zryfish